### DISCUSSION TITLE
[question] Could you please provide the cmake project?

### DISCUSSION THREAD

**Discussion by zh1an** [Tencent/MMKV#765] [Q&A] [answered]

**Comment by lingol** [comment]
CMake project of what?

**Comment by zh1an** [comment]
> Cmake project of what?

@lingol 

`Vcpkg` helps you manage C and C++ libraries on Windows, Linux and MacOS. I want to add `mmkv` project to `vcpkg`, but now, building `mmkv` via `vcpkg` is too complex. CMake project will be helpful to do the above.

**Comment by lingol** [comment, marked as answer]
Assuming you are talking about build MMKV on Win32, no we don't provide a CMake project because the VS2015 doesn't support CMake. Some of our largest users (*cough* WeChat for Windows *cough*) still use VS2015 to build their products.

You can make it yourself by looking at the CMake project of the POSIX port of MMKV.
https://github.com/Tencent/MMKV/tree/master/POSIX

**Comment by lingol** [comment]
It's been added long ago.
